build.sbt

```diff
@@ -8,7 +8,8 @@ scalaVersion := "2.13.10"
 crossScalaVersions := Seq(scalaVersion.value, "2.12.17", "3.2.1")
 
 libraryDependencies ++= Seq(
-    "org.asynchttpclient" % "async-http-client" % "2.12.3",
+    "org.asynchttpclient" % "async-http-client" % "2.12.3" exclude("io.netty", "netty-codec"),
+    "io.netty" % "netty-codec" % "4.1.68.Final",
```

Comment:
Nice solution, gonna use that for another similar dependency 👍 

The latest version of netty - as of yesterday! - seems to be 4.1.90. Should we bump to that version at the same time?